### PR TITLE
UG Chap. 1 updated

### PR DESCRIPTION
**Pull request by takugoishii**
UG Sec. 1.1 temporarily completed with original styles.

### CODE REVIEW

File: localized/ja/UserGuide-chapter1.adoc [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+:awestruct-description: Chapter 1. OptaPlanner イントロダクション
+:awestruct-layout: localizedBase
+:awestruct-lang: jp
+:awestruct-priority: 1.0
+:showtitle:
+xxx
+
+////
+
+Failed, to be deleted soon >>>>>
```
Comment (ge0ffrey) [inline localized/ja/UserGuide-chapter1.adoc]: I presume a second PR is coming to delete these comments? I'll merge the current PR already.

Thank you for contributing this translation! Great work.
If you want to be recognized as a contributor, please submit a PR to add yourself in this file:
  https://github.com/droolsjbpm/optaplanner-website/blob/master/_config/users.yml

**Comment by ge0ffrey** [comment]
Note: If you want to efficiently translate the images too (not that I am expecting that, keeping them in English is fine too), this can be done quickly and easily:

- Don't just edit the PNG files with Paint - that's a bad idea.
- Instead, open the SVG source file of an image with InkScape (open source vector graphics app from inkscape.org), use the text tool to change the English text to Japanese text and then do menu File -> export bitmap to PNG.

The SVG source images are here:
  https://github.com/droolsjbpm/optaplanner/tree/master/optaplanner-docs/src/main/docbook/en-US/images/Chapter-Planner_introduction

File: localized/ja/UserGuide-chapter1.adoc (hunk 1 above)
Comment (takugoishii) [inline localized/ja/UserGuide-chapter1.adoc]: Dear Geoffrey, 

Thank you for your managing the Japanese localized files. 
Also, thank you for the information on how to change the text placed in images based on the SVG format. I will try that. 

I believe the quality of the present translation would well be accepted, hoping more sections are collaboratively added to achieve our first goal. 

To tell the truth, the PR I sent was a result of my careless operation failure I made as a beginner while I was browsing around the related GitHub website. Actually, I had to keep the version within our Japanese branch, which will continuously be maintained by Kobayashi-san. The final version will be prepared for the upstream PR by him after collecting and unifying all the files created by local contributors. In that process, the needless comments you found in the temporary file will also be deleted. 

—

This is a super tiny correction in the German index.html I found in the localization folder today: 
In sentence 4: Produkte un Dienste —> Produkte und Dienste

Best regards, 
Takugo Ishii

Comment (ge0ffrey) [inline localized/ja/UserGuide-chapter1.adoc]: CC @tkobayas
Thank you, Ishii-san. This is indeed a good start, a very nice step forward to the old situation (with any translations). 

German index.html fixed.

**Comment by takugoishii** [comment]
Hello Geoffrey, 

This is a proposal for handling text in the SVG images for OptaPlanner User Guide. 

Referring to the description in your previous email, I briefly checked all SVG files stored on GitHub for Chapter 1, and found that you didn't include text in them. So, I tentatively created the SVG files with text. Please have a look at a set of sample image files in the zip file attached. 

- en
  - compatibility_original.svg: your present English version
  - compatibility.svg: modified with text added
- ja
  - compatibility.svg: Japanese text replaced based on the English version

For English, I used Arial font, which is the font you use for Gothic as I confirmed in the SVG tags. I used Affinity Designer, which is an Illustrator-alternative application running on my favorite Mac, because InkScape could not start even on the latest X11. (I use the OS X El Capitan Public Beta, which might be problematic.) However, I expect you can smoothly open and re-edit the SVG with your InkScape. 

The SVG with text (especially the English version) can be the template for multilingual translation. 

How about this? If you have any requests or suggestions, please let me know. 
If you would agree with going on this line, I want to similarly proceed with other images as well in the present translation work. (Anyway, for Japanese version, I need to add text.) 

Regards, 
Takugo Ishii

**Comment by ge0ffrey** [comment]
Hi Takugo-san
    
      With kind regards,
        Geoffrey De Smet
    
    On 16/09/15 16:49, Takugo wrote:
    
    Hello Geoffrey, 
      
      This is a proposal for handling text in the SVG images for
      OptaPlanner User Guide. 
      
      Referring to the description in your previous email, I briefly
      checked all SVG files stored on GitHub for Chapter 1, and found
      that you didn't include text in them.
    
    I did, but I fear the English text might not show up on your machine
    because of the font type being set to "Arial":
    I've seen a similar problem on other people's machines too.
    If this is the problem, there are 2 ways to workaround it:
    - Install the MS Core Fonts on linux:
    http://mscorefonts2.sourceforge.net/ (this is legal - MS released
    them in the public domain in 1999)
    - Open the SVG file as an xml file and replace "Arial" with
    "Liberation Sans"
    
    But reading the rest of this mail, it might due to using a different
    SVG editor (which is ok of course).
    
     So, I tentatively created the SVG files with text.
      Please have a look at a set of sample image files in the zip file
      attached. 
    
    I didn't get the attachment, feel free to mail directly to me
    (gds.geoffrey.de.smet a t gmail.com)
    
      
      - en
      - compatibility_original.svg: your present English version
      - compatibility.svg: modified with text added
      - ja
      - compatibility.svg: Japanese text replaced based on the English
      version
    
    For the website, I recommend to put the only the
    "${originalName}_ja.svg" and "${originalName}_ja.png" in the PR.
    The english SVG's should stay in the optaplanner-docs only, so
    there's no doubt that the canonical location of the English
    originals are.
    
    Because the Japanese SVG's might be useful later on, I believe we
    need to keep them around to, so just put them next to the japanese
    png's on optaplanner-website.
    To clearly map the English originals with the Japanese derivatives
    (which is useful for future maintenance),
    I recommend to use the same filename as the English, but suffixed
    with "_ja".
    
      
      For English, I used Arial font, which is the font you use for
      Gothic as I confirmed in the SVG tags. I used Affinity Designer,
      which is an Illustrator-alternative application running on my
      favorite Mac, because InkScape could not start even on the latest
      X11. (I use the OS X El Capitan Public Beta, which might be
      problematic.) However, I expect you can smoothly open and re-edit
      the SVG with your InkScape. 
    
    Sounds good - whatever works best for you and your machine :)
    
      
      The SVG with text (especially the English version) can be the
      template for multilingual translation. 
    
    The originals have English text (open them with a text editor as
    proof) - I'd rather figure out why it doesn't show up on your
    machine in your editor and fix the originals.
    
      
      How about this? If you have any requests or suggestions, please
      let me know. 
      If you would agree with going on this line, I want to similarly
      proceed with other images as well in the present translation work.
      (Anyway, for Japanese version, I need to add text.) 
    
    
    Thanks for all your work. I look forward to the Japanese versions of
    those images.
    If you would like to be recognized for your contributions on the
    team page:
      http://www.optaplanner.org/community/team.html
    then send in a PR to add yourself in this UTF-8 file:
     
https://github.com/droolsjbpm/optaplanner-website/blob/master/_config/users.yml
    
    
      
      Regards, 
      Takugo Ishii
      
      
      
      > 2015/09/08 16:36、Geoffrey De Smet
      <notifications@github.com> のメール：
      > 
      > Note: If you want to efficiently translate the images too
      (not that I am expecting that, keeping them in English is fine
      too), this can be done quickly and easily:
      > 
      > • Don't just edit the PNG files with Paint - that's a bad
      idea.
      > • Instead, open the SVG source file of an image with InkScape
      (open source vector graphics app from inkscape.org), use the text
      tool to change the English text to Japanese text and then do menu
      File -> export bitmap to PNG.
      > The SVG source images are here:
      >
https://github.com/droolsjbpm/optaplanner/tree/master/optaplanner-docs/src/main/docbook/en-US/images/Chapter-Planner_introduction
      > 
      > —
      > Reply to this email directly or view it on GitHub.
      
      
      
      —
        Reply to this email directly or view
          it on GitHub.

**Comment by takugoishii** [comment]
Dear Geoffrey, 

I understood that the missing text in SVG files was likely a problem of software incompatibility. I may install InkScape on Ubuntu to check your original files. 

Please find the archive file I separately sent to your private email address as you suggested. (Previously, I attached it to the reply.gituhub.com email, but might not reach you.) I included compatibility.svg, which is generated by my graphic software. If the appearance of the English and Japanese SVG files is identical to that of PNG files, I may continuously use the software to further create Japanese files. If the text does not appear correctly, I may try to create Japanese files using InkScape on Linux as you did in order to avoid other trials for unknown better solutions. 


> The originals have English text (open them with a text editor as
> proof) - I'd rather figure out why it doesn't show up on your
> machine in your editor and fix the originals.


I additionally checked on another Mac drawing software iDraw, its latest version, but the result was nearly the same as that obtained by Affinity Designer I used initially. Text is missing. In this case, color was not perfectly reproduced at least in one file. (I wonder if the problem was caused by color profile setting, but don't want to think about it further now.) I also briefly checked on Illustrator CS5 (a bit old version). Concerning text, it is not displayed, neither. In some files, layout is also collapsed badly. 
So, the SVG looks unstable from the point of view of text compatibility. If I simply say based on the number of software checked, InkScape could be an exceptional case. It happened that only Affinity Designer worked correctly for import and export at least my OS X system is concerned at the moment. 


> - Open the SVG file as an xml file and replace "Arial" with "Liberation Sans"


Failed, neither. Additionally, “Helvetica,” “Times New Roman,” “Times,” “Courier,” and “Osaka” (long-living previous OS X system font in the Japanese language setting) all failed. 
This problem looks originating from software incompatibility. 


Best regards, 
Takugo

**Comment by ge0ffrey** [comment]
Hi Takugo-san,
    
    My apologies for the late reply.
    The japanese png images look good.
    Let's keep your japanese SVG sources along side them so the sources
    aren't lost.
    As for the english SVG sources that only work well in Inkscape:
    I 'll see if anything can be done about that, I recently upgraded
    Inkscape version, but most svg's haven't been touched since I
    upgraded.
    
    If you can submit a Pull Request which adds the japanese png and svg
    images and includes the png images in the adoc, I 'll review, merge
    and publish optaplanner-website.
    Thanks for your contributions.
    
      With kind regards,
        Geoffrey De Smet
    
    On 18/09/15 10:40, Takugo wrote:
    
    Dear Geoffrey, 
      
      I understood that the missing text in SVG files was likely a
      problem of software incompatibility. I may install InkScape on
      Ubuntu to check your original files. 
      
      Please find the archive file I separately sent to your private
      email address as you suggested. (Previously, I attached it to the
      reply.gituhub.com email, but might not reach you.) I included
      compatibility.svg, which is generated by my graphic software. If
      the appearance of the English and Japanese SVG files is identical
      to that of PNG files, I may continuously use the software to
      further create Japanese files. If the text does not appear
      correctly, I may try to create Japanese files using InkScape on
      Linux as you did in order to avoid other trials for unknown better
      solutions. 
      
      
      > The originals have English text (open them with a text editor
      as
      > proof) - I'd rather figure out why it doesn't show up on your
      > machine in your editor and fix the originals.
      
      
      I additionally checked on another Mac drawing software iDraw, its
      latest version, but the result was nearly the same as that
      obtained by Affinity Designer I used initially. Text is missing.
      In this case, color was not perfectly reproduced at least in one
      file. (I wonder if the problem was caused by color profile
      setting, but don't want to think about it further now.) I also
      briefly checked on Illustrator CS5 (a bit old version). Concerning
      text, it is not displayed, neither. In some files, layout is also
      collapsed badly. 
      So, the SVG looks unstable from the point of view of text
      compatibility. If I simply say based on the number of software
      checked, InkScape could be an exceptional case. It happened that
      only Affinity Designer worked correctly for import and export at
      least my OS X system is concerned at the moment. 
      
      
      > - Open the SVG file as an xml file and replace "Arial" with
      "Liberation Sans"
      
      
      Failed, neither. Additionally, “Helvetica,” “Times New Roman,”
      “Times,” “Courier,” and “Osaka” (long-living previous OS X system
      font in the Japanese language setting) all failed. 
      This problem looks originating from software incompatibility. 
      
      
      Best regards, 
      Takugo
      
      
      
      
      > 2015/09/17 22:29、Geoffrey De Smet
      <notifications@github.com> のメール：
      > 
      > 
      > 
      > 
      > 
      > 
      > Hi Takugo-san
      > 
      > With kind regards,
      > Geoffrey De Smet
      > 
      > On 16/09/15 16:49, Takugo wrote:
      > 
      > Hello Geoffrey, 
      > 
      > This is a proposal for handling text in the SVG images for
      > OptaPlanner User Guide. 
      > 
      > Referring to the description in your previous email, I
      briefly
      > checked all SVG files stored on GitHub for Chapter 1, and
      found
      > that you didn't include text in them.
      > 
      > I did, but I fear the English text might not show up on your
      machine
      > because of the font type being set to "Arial":
      > I've seen a similar problem on other people's machines too.
      > If this is the problem, there are 2 ways to workaround it:
      > - Install the MS Core Fonts on linux:
      > http://mscorefonts2.sourceforge.net/ (this is legal - MS
      released
      > them in the public domain in 1999)
      > - Open the SVG file as an xml file and replace "Arial" with
      > "Liberation Sans"
      > 
      > But reading the rest of this mail, it might due to using a
      different
      > SVG editor (which is ok of course).
      > 
      > So, I tentatively created the SVG files with text.
      > Please have a look at a set of sample image files in the zip
      file
      > attached. 
      > 
      > I didn't get the attachment, feel free to mail directly to me
      > (gds.geoffrey.de.smet a t gmail.com)
      > 
      > 
      > - en
      > - compatibility_original.svg: your present English version
      > - compatibility.svg: modified with text added
      > - ja
      > - compatibility.svg: Japanese text replaced based on the
      English
      > version
      > 
      > For the website, I recommend to put the only the
      > "${originalName}_ja.svg" and "${originalName}_ja.png" in the
      PR.
      > The english SVG's should stay in the optaplanner-docs only,
      so
      > there's no doubt that the canonical location of the English
      > originals are.
      > 
      > Because the Japanese SVG's might be useful later on, I
      believe we
      > need to keep them around to, so just put them next to the
      japanese
      > png's on optaplanner-website.
      > To clearly map the English originals with the Japanese
      derivatives
      > (which is useful for future maintenance),
      > I recommend to use the same filename as the English, but
      suffixed
      > with "_ja".
      > 
      > 
      > For English, I used Arial font, which is the font you use for
      > Gothic as I confirmed in the SVG tags. I used Affinity
      Designer,
      > which is an Illustrator-alternative application running on my
      > favorite Mac, because InkScape could not start even on the
      latest
      > X11. (I use the OS X El Capitan Public Beta, which might be
      > problematic.) However, I expect you can smoothly open and
      re-edit
      > the SVG with your InkScape. 
      > 
      > Sounds good - whatever works best for you and your machine :)
      > 
      > 
      > The SVG with text (especially the English version) can be the
      > template for multilingual translation. 
      > 
      > The originals have English text (open them with a text editor
      as
      > proof) - I'd rather figure out why it doesn't show up on your
      > machine in your editor and fix the originals.
      > 
      > 
      > How about this? If you have any requests or suggestions,
      please
      > let me know. 
      > If you would agree with going on this line, I want to
      similarly
      > proceed with other images as well in the present translation
      work.
      > (Anyway, for Japanese version, I need to add text.) 
      > 
      > 
      > Thanks for all your work. I look forward to the Japanese
      versions of
      > those images.
      > If you would like to be recognized for your contributions on
      the
      > team page:
      > http://www.optaplanner.org/community/team.html
      > then send in a PR to add yourself in this UTF-8 file:
      > 
      >
https://github.com/droolsjbpm/optaplanner-website/blob/master/_config/users.yml
      > 
      > 
      > 
      > Regards, 
      > Takugo Ishii
      > 
      > 
      > 
      > > 2015/09/08 16:36、Geoffrey De Smet
      > <notifications@github.com> のメール：
      > > 
      > > Note: If you want to efficiently translate the images
      too
      > (not that I am expecting that, keeping them in English is
      fine
      > too), this can be done quickly and easily:
      > > 
      > > • Don't just edit the PNG files with Paint - that's a
      bad
      > idea.
      > > • Instead, open the SVG source file of an image with
      InkScape
      > (open source vector graphics app from inkscape.org), use the
      text
      > tool to change the English text to Japanese text and then do
      menu
      > File -> export bitmap to PNG.
      > > The SVG source images are here:
      > >
      >
https://github.com/droolsjbpm/optaplanner/tree/master/optaplanner-docs/src/main/docbook/en-US/images/Chapter-Planner_introduction
      > > 
      > > —
      > > Reply to this email directly or view it on GitHub.
      > 
      > 
      > 
      > —
      > Reply to this email directly or view
      > it on GitHub.
      > —
      > Reply to this email directly or view it on GitHub.
      > 
      
      —
        Reply to this email directly or view
          it on GitHub.

**Pull request merged**